src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__product-title {
+    margin-block: 40px 0;
+    font-family: $main-font-family;
```

Comment:
The variable `$main-font-family` is used here. Ensure that this variable is defined somewhere in your SCSS files, possibly in a variables file or at the top of this file.